plasma_framework/docs/design/add-new-tx-type-check-lists.md

```diff
@@ -0,0 +1,9 @@
+# Check List For Adding New Transaction Types
+
+This document provides some check lists that whenever we are designing a new transaction type. Since there are some known restriction of the Plasma Framework design, please make sure that new tx type and exit game does not break the global assumptions and limitations.
+
+1. Aside from deposit transactions, all transactions should be unique. This promises the `OutputId` to be unique as `OutputId` uniqueness is a global assumption placed to `PlasmaFramework`.
+1. output index is max to `TX_OFFSET`  (10000) of the `UtxoPos` schema. As a result, no more than such amounts of inputs or outputs in one transaction.
```

Comment:
Start the first word with an upper case.

Reply:
actually I think we can have more outputs than `TX_OFFSET`, only number of outputs are strictly limited.

Reply:
> actually I think we can have more outputs than TX_OFFSET

Do you mean inputs? hmmm...I think you're right on that

Reply:
You are right - _inputs_. Corrected it.